src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -792,6 +802,15 @@ describe("Artwork type", () => {
           })
         })
 
+        it("is not hangable if the category is not applicable to wall display like installations", () => {
+          artwork.category = "installation"
+          artwork.width = 100
+          artwork.height = 100
+          return runQuery(query, rootValue).then(data => {
+            expect(data.artwork.is_hangable).toBe(false)
+          })
+        })
```

Comment:
I added this test to see whether the logic in `is_hangable` was doing what we thought it was, and it failed at first